src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -138,17 +134,33 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("A space seperated list of functions to load.")
                 .Callback(f => EnabledFunctions = f);
 
-            return base.ParseArgs(args);
+            Parser
+                .Setup<bool>("verbose")
+                .WithDescription("When false, hides system logs other than warnings and errors.")
+                .SetDefault(false)
+                .Callback(v => VerboseLogging = v);
+
+            var parserResult = base.ParseArgs(args);
+            bool verboseLoggingArgExists = parserResult.UnMatchedOptions.Where(o => o.LongName.Equals("verbose", StringComparison.OrdinalIgnoreCase)).Any();
```

Comment:
We can probably just do `parserResult.UnMatchedOptions.Any(o => o.LongName.Equals("verbose", StringComparison.OrdinalIgnoreCase))` I think.

Reply:
done!